src/components/DeploymentForm.vue

```diff
@@ -119,14 +134,20 @@
   const { value: workPoolName } = useField<string | null>('workPoolName')
   const { value: workQueueName } = useField<string | null>('workQueueName')
   const { value: tags } = useField<string[] | null>('tags')
+  const { value: infrastructureOverrides } = useField<string>('infrastructureOverrides')
 
   const emit = defineEmits<{
     (event: 'submit', value: DeploymentUpdate): void,
     (event: 'cancel'): void,
   }>()
 
   const submit = handleSubmit((values) => {
-    emit('submit', values)
+    isJson(infrastructureOverrides.value)
```

Comment:
I've updated this @pleek91 but I don't like it as much from a UX perspective as there's no alert to tell the user the json is invalid.  Any thoughts on the best way to do that?

Reply:
Ohh.... sorry I think I know.  Ignore

Reply:
😞 Sorry @pleek91 - should be updated now